packages/shared/src/components/tooltips/LazyTooltip.module.css

```diff
@@ -0,0 +1,159 @@
+@keyframes fade-in-top {
```

Comment:
General question about this file, should we maybe opt to use tailwind @apply?
Just so we can generalize CSS a bit?

And maybe if it's possible we can add some of these classes directly to the elements?

Reply:
Unfortunately, we can't add the classes directly to the elements since we have to set the value from 0% to 100% explicitly.

Reply:
I tried using `@apply-1` but I'm not sure why it is throwing an error.